package/MDAnalysis/coordinates/base.py

```diff
@@ -1741,7 +1749,66 @@ def get_aux_descriptions(self, auxnames=None):
             auxnames = self.aux_list
         descriptions = [self._auxs[aux].get_description() for aux in auxnames]
         return descriptions
-
+    
+    @property
+    def transformations(self):
+        """ Returns the list of transformations"""
+        return self._transformations
+    
+    @transformations.setter
+    def transformations(self, transform_list):
+        if not self._transformations:
+            self._transformations = transform_list
+        else:
+            raise ValueError("Transformations are already set")
+        
+    def add_transformations(self, transform_list):
+        """ Add all transformations to be applied to the trajectory.
+        
+        
+        The transformations can be part of the :mod:`~MDAnalysis.transformations` 
+        module, or created by the user, and are stored as a list `transformations`. 
+        This list can only be modified once, and further calls of this function will
+        raise an exception.
+        Transformation that are part of the :mod:`~MDAnalysis.transformations` module 
+        may or may not require arguments. 
+        e. g. translate the coordinates by a (1,1,1) vector:
```

Comment:
These are not the arguments to document. The question here is what arguments will be given to the transformations by the reader: i.e. a TimeStep.

The vector you provide to the `translate` transformation is an implementation detail.